PSKoans/Public/Show-Advice.ps1

```diff
@@ -16,6 +16,28 @@
         Get-Content |
         ConvertFrom-Json
 
+        # Checking for errors
+        if (-not $AdviceObject) {
+            $ErrorDetails = @{
+                ExceptionType    = 'System.IO.FileNotFoundException'
+                ExceptionMessage = 'Could not find any advices that match the specified Name'
+                ErrorId          = 'PSKoans.NoMatchingAdvicesFound'
```

Comment:
```suggestion
                ErrorId          = 'PSKoans.NoMatchingAdviceFound'
```

Reply:
I used [this](https://github.com/vexx32/PSKoans/blob/master/PSKoans/Public/Get-Karma.ps1#L55..L59) as a source, and it has `NoMatchingKoansFound` (plural) in its `ErrorId` so I thought it'd be better to keep the convention. Not a big deal though.